telebot/__init__.py

```diff
@@ -5889,100 +5890,111 @@ def _check_middleware(self, update_type):
 
     def _run_middlewares_and_handler(self, message, handlers, middlewares, update_type):
```

Comment:
And this function should be renamed to smth like _run_update_task

Reply:
Don't like this name. Let's left it for next updates.